types/submessages_test.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestReplySerialization(t *testing.T) {
+	// Deserializaton
```

Comment:
I guess we don't really care about deserialization here, so I'd just remove the comment. Or do you want to add a test anyways?

Reply:
Yeah this is from copy/paste of a different test. Will remove, good point

Reply:
updated